qa-test

```diff
@@ -25,18 +20,14 @@ test_typos()
 if [ "$1" = "--python" ]; then
   test_python
   exit 0
-elif [ "$1" = "--style" ]; then
-  test_style
-  exit 0
-elif [ "$1" = "--unit" ]; then
-  test_unit
+elif [ "$1" = "--quick" ]; then
+  test_quick
   exit 0
 elif [ "$1" = "--typos" ]; then
   test_typos
   exit 0
 else
   test_python
-  test_style
-  test_unit
+  test_quick
```

Comment:
I don't think tests requiring specific dependencies should be run by default qa-test ( which runs at every merge)
It could be s separate step on the Jenkinsfile though